cmd/soroban-cli/src/utils/contract-template/src/lib.rs

```diff
@@ -4,6 +4,12 @@ use soroban_sdk::{contract, contractimpl, vec, Env, String, Vec};
 #[contract]
 pub struct Contract;
 
+// This is a sample contract. Replace this placeholder with your own contract logic.
+// A corresponding test example is available in `test.rs`.
+//
+// For comprehensive examples, visit <https://github.com/stellar/soroban-examples>.
```

Comment:
I think we should also link stellar docs' tutorial here 
https://developers.stellar.org/docs/build/smart-contracts

Reply:
Good call. Just added it.